packages/protocol/src/node/server.ts

```diff
@@ -331,9 +331,9 @@ export class Server {
 		return stringify(value, undefined, (p) => this.storeProxy(p));
 	}
 
-	private getProxy(proxyId: number | Module): ProxyData {
+	private getProxy(proxyId: number | Module, method: string): ProxyData {
```

Comment:
Should we really pass a value for this function, even if it's only going to be used as a message on the error?

Reply:
Yeah, it would have been better to just use `trace` logging to figure out why that error was occurring. Will revert.